resources/vlanresource_test.go

```diff
@@ -39,25 +39,25 @@ type vlanRsrcValidator struct {
 func (vt *vlanRsrcValidator) nextCfgState() {
 	vt.expCfg = vt.expCfg[1:]
 	if len(vt.expCfg) > 0 {
-		log.Printf("after pop cfg is: %+v\n", vt.expCfg[0])
+		log.Debugf("after pop cfg is: %+v\n", vt.expCfg[0])
```

Comment:
yeah, we can remove these if they become excessive. I had added them to debug the tests in these files but left them there for future tests.